adapters/adkernel/adkernel.go

```diff
@@ -15,6 +16,14 @@ import (
 	"github.com/prebid/prebid-server/v2/openrtb_ext"
 )
 
+const (
+	mf_suffix        = "__mf"
+	mf_suffix_banner = "b" + mf_suffix
+	mf_suffix_video  = "v" + mf_suffix
+	mf_suffix_audio  = "a" + mf_suffix
```

Comment:
as per `static/bidder-info/adkernel.yaml` audio format is not supported. Should remove `mf_suffix_audio` or update `static/bidder-info/adkernel.yaml` to support audio format

Reply:
@onkarvhanumante done